src/components/Phones/Phones.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import { getPhones } from '../../api/api';
+import { BreadCrumbs } from '../BreadCrumbs';
+import { ProductsList } from '../ProductsList';
+import {
+  ProductsLoader2x,
+} from '../ProductsLoader/ProductsLoader2x';
+
+import './phones.scss';
+
+export const Phones: React.FC = () => {
+  const [phonesList, setPhonesList] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+    getPhones(setPhonesList)
+      .then(() => setIsLoaded(true));
```

Comment:
it would be better to rewrite all similar API functions to standart way
```suggestion
    getPhones()
      .then((data) => setPhonesList(data))
      .catch((err) => // show error)
      .finally((data) => setIsLoaded(true));
```